src/node/desktop/src/ui/utils.ts

```diff
@@ -60,8 +60,13 @@ function normalizeSeparators(path: string, separator = '/') {
  * @param {string} path
  * @return {*} 
  */
-export function normalizeSeparatorsNative(path: string) {
+export function normalizeSeparatorsNative(path: string, forcedSeparator = '') {
```

Comment:
Why not just export and use the `normalizeSeparators()` function above? This function doesn't need to be changed.

Reply:
Just didn't wanted another function being exported but your option is really the best